ironfish/src/consensus/verifier.ts

```diff
@@ -170,7 +170,7 @@ export class Verifier {
     tx?: IDatabaseTransaction,
   ): Promise<VerificationResult> {
     return this.chain.db.withTransaction(tx, async (tx) => {
-      const noteSize = await this.chain.notes.size(tx)
+      const noteSize = await this.chain.nullifiers.size(tx)
```

Comment:
This should be renamed to something like `nullifierSize` or `treeSize`. Otherwise, looks good

Reply:
Fixed